src/panels/config/devices/device-detail/ha-device-entities-card.ts

```diff
@@ -115,6 +130,28 @@ export class HaDeviceEntitiesCard extends LitElement {
 
   private _toggleShowDisabled() {
     this.showDisabled = !this.showDisabled;
+    if (!this.showDisabled || this._extDisabledEntityEntries !== undefined) {
```

Comment:
probably not something we will run into now, but if `entities` is changed we should also refetch `_extDisabledEntityEntries`